src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -62,7 +62,8 @@ class F extends AllFeaturesMaxLimitsGPUTest {
     texture: GPUTexture,
     textureViewDescriptor?: GPUTextureViewDescriptor
   ): GPURenderPassColorAttachment {
-    const view = texture.createView(textureViewDescriptor);
+    const { bindTextureResource = false } = this.params as { bindTextureResource?: boolean };
```

Comment:
I think I might be better to just pass in `bindTextureResource`? This is not type safe. If the parameters change, this will still compile, it will just always be false.

Reply:
I did that because `textureViewDescriptor` was optional. Maybe I should update `getColorAttachment` parameters with something like `getColorAttachment(texture, {textureViewDescriptor, bindTextureResource})` instead...

Reply:
@greggman I've fixed this in latest commit.